banking-service.py

```
@@ -9,6 +9,7 @@
 from datetime import datetime
 from pydantic import BaseModel
 import os
+from config import *
```

Ruff (0.11.9)

`from config import *` used; unable to detect undefined names

(F403)